### PR TITLE
[Feature Request] [MO] add ONNX operator LayerNorm support into frontend/onnx

### PR DESCRIPTION
**Pull request by SakuraYM**
### Details:
Add onnx op LayerNorm support in frontend/onnx
Add layer_norm test in layer_tests
Add layer_norm.prototxt into src/core/tests/models/onnx/
Add layer_norm test module into src/frontends/tests/onnx/onnx_import.in.cpp

### Tickets:
https://github.com/openvinotoolkit/openvino/issues/11914

### CODE REVIEW

**Review by tomdol — APPROVED** [review]
Overall I'm ok with this PR, we can test if switching to ONNX function expander can do the same thing later.

File: tests/layer_tests/pytest.ini
Comment (tomdol) [inline tests/layer_tests/pytest.ini:6]: @jiwaszki do you think those settings are needed?

Comment (jiwaszki) [reply tests/layer_tests/pytest.ini:6]: @SakuraYM please explain why these setting are required for this PR. I assume that these change MO test suite behavior, do tests still passing without them?

**Review by jiwaszki — CHANGES_REQUESTED** [review]

File: tests/layer_tests/pytest.ini
Comment (SakuraYM) [reply tests/layer_tests/pytest.ini:6]: sorry, It's not necessary, we can successfully run test without this code.
I'll delete it and submit a new commit

Comment (SakuraYM) [reply tests/layer_tests/pytest.ini:6]: > @SakuraYM please explain why these setting are required for this PR. I assume that these change MO test suite behavior, do tests still passing without them?

sorry, It's not necessary, we can successfully run test without this code.
I'll delete it and submit a new commit

**Review by jiwaszki — APPROVED** [review]

**Comment by SakuraYM** [comment]
Hi @tomdol , may I ask you a question? It's almost one month after you approved changes why **cj/jenkins** status is still expected?

**Comment by tomdol** [comment]
There have been problems with our CI. I've merged your PR with master and left the auto-merge enabled. We just need all of the current pipelines to finish spinning.

**Comment by SakuraYM** [comment]
After the last merge operation, my three cpp tests all failed. Does it need to be modified  ? or is it still just caused by the CI error and I'll just wait for the auto-merge?  : )

**Comment by SakuraYM** [comment]
hi @tomdol ,I have a question. 
Why all tests could pass locally ,but the CI logs show them fail because of
 “ C++ exception with description "Error during import of ONNX model expected to be in file: ./test_model_zoo/onnx/layer_norm_dynamic_4d.onnx. Could not open the file." thrown in the test body. ”?

Thanks!

**Comment by tomdol** [comment]
@ilya-lavrenov could you please have a look? I have a lot of trouble making the CI checks pass.

File: tests/layer_tests/onnx_tests/test_layernorm.py
Comment (rkazants) [inline tests/layer_tests/onnx_tests/test_layernorm.py:91]: please add at least one test into the pre-commit to avoid breaking.

File: tests/layer_tests/onnx_tests/test_layernorm.py
Comment (rkazants) [inline tests/layer_tests/onnx_tests/test_layernorm.py:100]: please remove it

File: tests/layer_tests/onnx_tests/test_layernorm.py
Comment (rkazants) [inline tests/layer_tests/onnx_tests/test_layernorm.py:21]: useless comments - please remove it

File: tests/layer_tests/onnx_tests/test_layernorm.py
Comment (rkazants) [inline tests/layer_tests/onnx_tests/test_layernorm.py:35]: please remove it.

**Review by rkazants — CHANGES_REQUESTED** [review]
print stuff remained in the tests.

File: src/frontends/onnx/frontend/src/op/layer_normalization.cpp
Comment (rkazants) [inline src/frontends/onnx/frontend/src/op/layer_normalization.cpp:82]: please avoid NGRAPH routines. Use only ov namespace

File: tests/layer_tests/onnx_tests/test_layernorm.py
Comment (SakuraYM) [reply tests/layer_tests/onnx_tests/test_layernorm.py:91]: line97-100 is the test case

Comment (SakuraYM) [reply tests/layer_tests/onnx_tests/test_layernorm.py:91]: > please add at least one test into the pre-commit to avoid breaking.

line97-100 is the test case

File: tests/layer_tests/onnx_tests/test_layernorm.py
Comment (SakuraYM) [reply tests/layer_tests/onnx_tests/test_layernorm.py]: This comment is to align other op comment formats

File: src/frontends/onnx/frontend/src/op/layer_normalization.cpp
Comment (rkazants) [inline src/frontends/onnx/frontend/src/op/layer_normalization.cpp:12]: please use ov include headers. ngraph is deprecated.

File: tests/layer_tests/onnx_tests/test_layernorm.py
Comment (rkazants) [reply tests/layer_tests/onnx_tests/test_layernorm.py:91]: No, it is not in the pre-commit. Please add below:

```
    @pytest.mark.precommit
```

File: tests/layer_tests/onnx_tests/test_layernorm.py
Comment (rkazants) [inline tests/layer_tests/onnx_tests/test_layernorm.py:22]: please remove it. It is aligned with other tests but this comment is useless.

**Comment by akladiev** [comment]
This PR will be closed in 2 weeks in case of no activity.

**Comment by akladiev** [comment]
This PR was closed because it has been stalled for 2 week with no activity.

**Pull request closed without merging**
